rasa/core/events/__init__.py

```diff
@@ -1181,3 +1181,33 @@ def as_dict(self) -> Dict[Text, Any]:
 
     def apply_to(self, tracker: "DialogueStateTracker") -> None:
         tracker.reject_action(self.action_name)
+
+
+class SessionStarted(Event):
+    """Mark the beginning of a new conversation session."""
+
+    type_name = "session_started"
+
+    def __hash__(self) -> int:
+        return hash(32143124320)
+
+    def __eq__(self, other: Any) -> bool:
+        return isinstance(other, SessionStarted)
+
+    def __str__(self) -> Text:
+        return "SessionStarted()"
+
+    def as_story_string(self) -> None:
+        warnings.warn(
```

Comment:
should this be a `warnings.warn`? e.g. is the user responsible to fix this or is this rather something we need to fix if it happens?

Reply:
should be `logger.warning()`  yes